Makefile

```diff
@@ -3,6 +3,7 @@ DEPLOY_NODE = 192.168.1.91
 
 deps:
 	go get -u github.com/jteeuwen/go-bindata/...
+	go get github.com/boltdb/bolt/...
```

Comment:
Why?

Reply:
When 'go install', it will complain cannot find package "github.com/boltdb/bolt". Shouldn't it put here?

Reply:
No it should be fetches automatically by go get ..., remove it.